aten/src/ATen/native/native_functions.yaml

```diff
@@ -3445,7 +3445,8 @@
 - func: pinverse(Tensor self, float rcond=1e-15) -> Tensor
   variants: function, method
 
-- func: poisson_nll_loss(Tensor input, Tensor target, bool log_input, bool full, float eps, int reduction) -> Tensor
+- func: poisson_nll_loss(Tensor input, Tensor target, bool log_input=False, bool full=False, float eps=1e-08, int reduction=Mean) -> Tensor
```

Comment:
There's already a `poisson_nll_loss` bound to the `torch.nn.functional` namespace with the correct defaults:

https://github.com/pytorch/pytorch/blob/7cdfd86a727ce01e41f2af81931064fb664513dd/torch/nn/functional.py#L2651

And here's the C++ API entry for completeness:

https://github.com/pytorch/pytorch/blob/828a9dcc048610bb5b45884634566093920b34cc/torch/csrc/api/include/torch/nn/functional/loss.h#L702

So I'm not sure what this PR is actually trying to accomplish for the user?

Reply:
Note that `torch.poisson_nll_loss` is an undocumented function that, if anything, should be hidden from the user interface.

Reply:
I added a note to the description. The reason I added the defaults to the native functions declarations is so generated bindings get to make use of them. I agree that adding back `torch.poisson_nll_loss` manually to avoid breaking backwards compatibility is a good move and will do that soon.